.github/workflows/tests.yaml

```diff
@@ -250,6 +251,7 @@ jobs:
       run: make test.integration
       env:
         KONG_TEST_DISABLE_CERTMANAGER: "true"
+        KONG_PLUGIN_IMAGE_REGISTRY_CREDENTIALS: ${{ secrets.KONG_PLUGIN_IMAGE_REGISTRY_CREDENTIALS }}
```

Comment:
@programmer04 Can we document this secret somewhere, e.g. in https://github.com/Kong/team-k8s and explain how to obtain these and how/when to rotate them?

Reply:
Yes, I'm going to document everything, when the tests reach the final state

Reply:
Can we have an issue tracking this?

Reply:
The issue for this has been created https://github.com/Kong/gateway-operator/issues/465 during merge of this pr